### ISSUE TITLE
[Bug]: Annoying error that Esbonio server already running

### ISSUE THREAD

**Issue by oharboe** [vscode-restructuredtext/vscode-restructuredtext#368]
### Expected behavior

I expected no such superfluous/confusing error on startup of the GUI.

### Actual behavior

There are no other ill effects, but the log contains this error:

```
[Log - 2:41:09 PM] Starting Language Server
[Log - 2:41:09 PM] Error: Language client is not ready yet
[esbonio.lsp] Workspace root file://[deleted]
[esbonio.lsp] Sphinx Config {'version': None, 'conf_dir': '[deleted]', 'src_dir': '[deleted]', 'build_dir': '[deleted]/_build', 'builder_name': 'html'}
[esbonio.lsp] Config dir [deleted]
[esbonio.lsp] Src dir [deleted]
[esbonio.lsp] Build dir [deleted]/_build/html
[esbonio.lsp] Doctree dir [deleted]/_build/doctrees
[Error - 2:41:09 PM] Server initialization failed.
Error: command 'esbonio.server.configuration' already exists
	at _.registerCommand (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:94:109536)
	at Object.registerCommand (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:103:20898)
	at ExecuteCommandFeature.register (/home/oyvind/.vscode/extensions/lextudio.restructuredtext-180.0.0/node_modules/vscode-languageclient/lib/client.js:1650:53)
	at ExecuteCommandFeature.initialize (/home/oyvind/.vscode/extensions/lextudio.restructuredtext-180.0.0/node_modules/vscode-languageclient/lib/client.js:1630:14)
	at LanguageClient.initializeFeatures (/home/oyvind/.vscode/extensions/lextudio.restructuredtext-180.0.0/node_modules/vscode-languageclient/lib/client.js:2462:21)
	at /home/oyvind/.vscode/extensions/lextudio.restructuredtext-180.0.0/node_modules/vscode-languageclient/lib/client.js:2147:18
[Log - 2:41:09 PM] Error: command 'esbonio.server.configuration' already exists
[Error - 2:41:09 PM] Starting client failed
Error: command 'esbonio.server.configuration' already exists
	at _.registerCommand (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:94:109536)
	at Object.registerCommand (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:103:20898)
	at ExecuteCommandFeature.register (/home/oyvind/.vscode/extensions/lextudio.restructuredtext-180.0.0/node_modules/vscode-languageclient/lib/client.js:1650:53)
	at ExecuteCommandFeature.initialize (/home/oyvind/.vscode/extensions/lextudio.restructuredtext-180.0.0/node_modules/vscode-languageclient/lib/client.js:1630:14)
	at LanguageClient.initializeFeatures (/home/oyvind/.vscode/extensions/lextudio.restructuredtext-180.0.0/node_modules/vscode-languageclient/lib/client.js:2462:21)
	at /home/oyvind/.vscode/extensions/lextudio.restructuredtext-180.0.0/node_modules/vscode-languageclient/lib/client.js:2147:18
```

### Detailed steps

It doesn't always happen on start of Visual Studio code.

However when I click "restart on Sphinx[html] 4.4.0", I get the error above in the log.

### Extension information

v180.0.0

### Visual Studio Code information

1.60.0

### Operating system information

Ubuntu 21

### Python information

3.9.7

### Sphinx information

4.4.0

**Comment by lextm** [comment]
It's a side effect when you click the status bar and select a different sphinx/docutil option, as behind the scene two changes to the settings file occur. Two restart operations of esbonio can lead to this issue if they happen too close to each other.

The ultimate solution is to redesign the sphinx/docutil selection so it makes a single change to settings file. A short term workaround is to suppress one of the restart operations.

Need some further investigation.

**Comment by lextm** [comment]
Only resolved the most troublesome case in release 182.0.0. More patches are needed for other possible cases.
